smart-contract-verifier/smart-contract-verifier-proto/proto/v2/smart-contract-verifier.proto

```diff
@@ -218,3 +220,19 @@ message ListCompilerVersionsResponse {
   /// Compiler versions available
   repeated string compiler_versions = 1;
 }
+
+message LookupMethodsRequest {
+  string bytecode = 1;
+  string abi = 2;
+  string source_map = 3;
+  map<uint32, string> file_ids = 4;
+}
+
+message LookupMethodsResponse {
+  message Method {
+    string file_name = 1;
+    int32 file_offset = 2;
+    int32 length = 3;
```

Comment:
Nit: maybe change to `uint32`? In general they are completely compatible types; `uint` may just convey better sematical meaning